vividus-tests/src/main/resources/properties/environment/system/saucelabs/mobile_app/ios/environment.properties

```diff
@@ -1,6 +1,6 @@
-selenium.grid.appium-version=1.21.0
+selenium.grid.appium-version=1.22.0
```

Comment:
I would move this update to a separate commit and upgraded the version for Android system tests as well